aait/backend/group-2A/Application/Features/Like/Handlers/Queries/GetPostLikesQueryHandler.cs

```diff
@@ -16,10 +19,18 @@ public GetPostLikesQueryHandler(IUnitOfWork unitOfWork, IMapper mapper)
             _unitOfWork = unitOfWork;
         }
 
-        public async Task<List<UserDto>> Handle(GetPostLikesQuery request, CancellationToken cancellationToken)
+        public async Task<BaseCommandResponse<List<UserDto>>> Handle(GetPostLikesQuery request, CancellationToken cancellationToken)
         {
             var likes = await _unitOfWork.likeRepository.GetLikers(request.Id);
-            return _mapper.Map<List<UserDto>>(likes);
+
+            if (likes == null || likes.Count == 0)
+            {
```

Comment:
remove likes.count == 0 check. because empty list has meaning . it means just there no users who liked the post.